src/apps/competitions/tests/test_phase_migration.py

```diff
@@ -95,3 +93,9 @@ def test_all_submissions_migrated_before_changing_phase_status(self):
 
         mock_start = self.mock_migration()
         assert mock_start.call_count == 0
+
+    def test_only_parent_submissions_migrated(self):
+        self.parent_submission = self.make_submission()
```

Comment:
can we assert in here that many children exist to make sure we dont have just a parent and no children so we get a false positive?